dashboards/google-backupdr/README.md

```diff
@@ -3,4 +3,5 @@
 |BackupDr Monitoring|
 |:---------------------|
 |Filename: [backupdr-monitoring.json](backupdr-monitoring.json)|
-|This dashboard has 4 charts for the related [Backup DR metrics], including metrics for storage pool utilization, vdisk count utilization, OnVault pool Utilization along with Protected Data volume.|
+|This dashboard has 6 charts for the related [Backup DR metrics], including metrics for storage pool utilization, vdisk count utilization, OnVault pool Utilization, Protected 
```

Comment:
This link is not formatted correctly. We can either remove the brackets around Backup DR metrics so it is not a link or we can add a link that points to the metric list for the backup and dr service.

```suggestion
|This dashboard has 6 charts for the related [Backup DR metrics](https://cloud.google.com/monitoring/api/metrics_gcp#gcp-backupdr), including metrics for storage pool utilization, vdisk count utilization, OnVault pool Utilization, Protected 
```

Reply:
Done, removed brackets around Backup DR Metrics